calico/common.py

```diff
@@ -119,6 +127,29 @@ def validate_cidr(cidr, version):
         return False
 
 
+def canonicalise_cidr(cidr, version):
+    if cidr is None:
+        return None
+    nw = netaddr.IPNetwork(cidr, version=version)
+    return intern(str(nw))
+
+
+def validate_mac(mac):
+    try:
+        netaddr.EUI(mac, dialect=eui48.mac_unix)
+    except (netaddr.core.AddrFormatError, ValueError, TypeError):
+        return False
+    else:
+        return True
```

Comment:
Is there a reason we validate MAC address this way, rather than calling <code>netaddr.valid_mac(<em>mac</em>)</code>?